modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -2067,6 +2067,7 @@ CV__DNN_INLINE_NS_END
 
 #include <opencv2/dnn/layer.hpp>
 #include <opencv2/dnn/dnn.inl.hpp>
+#include <opencv2/dnn/utils/npy_blob.hpp>
```

Comment:
In general we don't import utilities by default with the module itself (e.g. `core/utility.hpp` or `dnn/shape_utils.hpp`).